### PR TITLE
fix: invert DNS logic

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Do the right thing #362

## Why?

Consent is inverted otherwise.

### CODE REVIEW

**Review by ashishpuliyel — APPROVED** [review]
Looks good to me

File: src/aus/getConsentState.ts [hunk 1]
```diff
@@ -6,6 +6,6 @@ export const getConsentState: () => Promise<AUSConsentState> = async () => {
 	const uspData = await getUSPData();
 
 	return {
-		personalisedAdvertising: uspData.uspString.charAt(2) === 'Y',
+		personalisedAdvertising: uspData.uspString.charAt(2) !== 'Y',
```
Comment (ioanna0) [inline src/aus/getConsentState.ts:9]: wondering if we could have keep the same name as CCPA which is negative `doNotSell` . Not blocking of course

**Review by ioanna0 — APPROVED** [review]

File: src/aus/getConsentState.ts (hunk 1 above)
Comment (mxdvl) [reply src/aus/getConsentState.ts:9]: The third character is either `Y`, `N` or `-`.
We want personalised advertising if it's either `N`, or `-`.

[As per the docs](https://github.com/InteractiveAdvertisingBureau/CCPA-reference-code):

> The usprivacy string as the format: `vnol` where
`v` = version (int)
`n` = Notice Given (char)
`o` = OptedOut (char)
`l` = Lspact (char)

Comment (mxdvl) [reply src/aus/getConsentState.ts:9]: I think it's better to have our code match our copy (ON / OFF &larr;&rarr; `true` / `false`)

Comment (mxdvl) [reply src/aus/getConsentState.ts]: I've clarified the logic in two steps with b86ee19.

**Pull request merged**